webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const webpack = require('webpack');
+
+module.exports = {
+	mode: 'development',
+	devServer: {
+		historyApiFallback: {
+			disableDotRule: true
+		}
+	},
+	entry: './src/index',
+	output: {
+		path: path.resolve(__dirname, 'build'),
+		publicPath: '/',
+		filename: 'bundle.js'
+	},
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				exclude: /node_modules/,
+				use: [ 'babel-loader' ]
+			}
+		]
+	},
+	plugins: [
+		new HtmlWebpackPlugin({
+			template: './src/index.html'
+		}),
+		new webpack.DefinePlugin({
+			PRODUCTION_API: JSON.stringify('https://octopus-bn-backend-staging.herokuapp.com//api/v1/')
```

Comment:
Again, @blaise82 , I think you mistakenly left a second slash after the **.com** in the Heroku link. Please double check.

Reply:
ohh let m e fix that